authentication-demo-service/pom.xml

```diff
@@ -4,7 +4,7 @@
 
 	
 	<groupId>io.mosip.testrig.authentication.demo</groupId>
-	<version>1.2.0.1-SNAPSHOT</version>
+	<version>1.2.1-java21-SNAPSHOT</version>

```

Comment:
If it is develop branch then version should be 1.2.1-SNAPSHOT

Reply:
Done.